### ISSUE TITLE
0.7.0-Beta runs tests in `assemble` task

### ISSUE THREAD

**Issue by ccjernigan** [Kotlin/kotlinx-kover#353]
**Describe the bug**
When updating from 0.7.0-Alpha to 0.7.0-Beta, I noticed that the `assemble` task is running my tests.

**Expected behavior**
Tests shouldn't run unless I invoke the "check" task

**Reproducer**
I have a branch on this project which reproduces the issue: https://github.com/zcash/secant-android-wallet/tree/kover-0.7.0-Beta

I made one test case intentionally fail, which makes the problem more obvious.  Kover is configured under build-conventions-secant, then applied to the individual modules.

1. Check out the repo with the kover-0.7.0-Beta branch
2. Run `./gradlew assemble`

**Reports**
```
 ./gradlew assemble

> Task :preference-api-lib:jvmTest FAILED

co.electriccoin.zcash.preference.model.entry.BooleanPreferenceDefaultTest[jvm] > value_default_true[jvm] FAILED
    java.lang.AssertionError at BooleanPreferenceDefaultTest.kt:19
```

**Environment**
 - Kover Gradle Plugin version:  0.7.0-Beta
 - Gradle version: 7.6
 - Kotlin project type: Kotlin/Multiplatform.  (The project linked above is an Android project, but it reproduces in a multiplatform module without any Android integration)

**Comment by shanshin** [comment]
Hi, thanks for the report!
Indeed, Kover artifacts should not be generated when running the `assemble` task.

**Comment by shanshin** [comment]
Fixed in `0.7.0`
